docs/BreakingChanges/118 Beginning in .NET 4.5, iPad is a browser capability.md

```diff
@@ -0,0 +1,28 @@
+## 118: Beginning in .NET 4.5, iPad is an identifier in the default ASP.NET browser capabilities file, so it should not be used in a custom capabilties file
```

Comment:
{
  "Id": "118",
  "Title": "Beginning in .NET 4.5, iPad is an identifier in the default ASP.NET browser capabilities file, so it should not be used in a custom capabilties file",
  "Details": "Beginning in .NET 4.5, iPad is an identifier in the default ASP.NET browser capabilities file, so it should not be used in a custom capabilties file",
  "Suggestion": "If iPad-specific capabilities are required, it is necessary to modify iPad behavior by setting capabilities on the pre-defined gateway refID \"IPad\" instead of by generating a new \"IPad\" ID by user agent matching.",
  "Link": null,
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/724085",
  "Notes": "For analyzer, can look for 'iPad' in .browser capability files\nSource analyzer status: Pri 3, Planned - No tracking work item yet",
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [ ] Not detectable via API analysis